### DISCUSSION TITLE
Google account not being able to verify uYou is safe when I log in

### DISCUSSION THREAD

**Discussion by themrsnoop** [qnblackcat/uYouPlus#678] [Q&A] [answered]
Every time I log into my account it shows this error, any help?
![00AA3A42-A8AD-4FA0-AFBA-EF17F280FD6D](https://user-images.githubusercontent.com/111259052/201928104-845e6c56-d133-4b30-9f73-ee6400d1c976.jpeg)

**Comment by qnblackcat** [comment, marked as answer]
https://github.com/qnblackcat/uYouPlus/wiki/FAQ
